diff -urN gnome-games-1.4.0.org/glines/Makefile.am gnome-games-1.4.0/glines/Makefile.am
--- gnome-games-1.4.0.org/glines/Makefile.am	Fri Mar 23 16:21:58 2001
+++ gnome-games-1.4.0/glines/Makefile.am	Fri Mar 23 16:22:27 2001
@@ -30,6 +30,6 @@
 install-data-local:
 	-$(mkinstalldirs) $(DESTDIR)$(scoredir)
 	touch $(DESTDIR)$(scoredir)/glines.scores
-	-chown games.games $(DESTDIR)$(scoredir)/glines.scores
+	-chgrp games $(DESTDIR)$(scoredir)/glines.scores
 	-chmod 664 $(DESTDIR)$(scoredir)/glines.scores
-	-chgrp games $(DESTDIR)$(bindir)/glines && chmod 2111 $(DESTDIR)$(bindir)/glines
+	-chgrp games $(DESTDIR)$(bindir)/glines && chmod 2755 $(DESTDIR)$(bindir)/glines
diff -urN gnome-games-1.4.0.org/gnibbles/Makefile.am gnome-games-1.4.0/gnibbles/Makefile.am
--- gnome-games-1.4.0.org/gnibbles/Makefile.am	Fri Mar 23 16:21:58 2001
+++ gnome-games-1.4.0/gnibbles/Makefile.am	Fri Mar 23 16:22:27 2001
@@ -96,27 +96,27 @@
 install-data-local:
 	-$(mkinstalldirs) $(DESTDIR)$(scoredir)
 	-touch $(DESTDIR)$(scoredir)/gnibbles.4.0.scores
-	-chown games.games $(DESTDIR)$(scoredir)/gnibbles.4.0.scores
+	-chgrp games $(DESTDIR)$(scoredir)/gnibbles.4.0.scores
 	-chmod 664 $(DESTDIR)$(scoredir)/gnibbles.4.0.scores
 	-touch $(DESTDIR)$(scoredir)/gnibbles.3.0.scores
-	-chown games.games $(DESTDIR)$(scoredir)/gnibbles.3.0.scores
+	-chgrp games $(DESTDIR)$(scoredir)/gnibbles.3.0.scores
 	-chmod 664 $(DESTDIR)$(scoredir)/gnibbles.3.0.scores
 	-touch $(DESTDIR)$(scoredir)/gnibbles.2.0.scores
-	-chown games.games $(DESTDIR)$(scoredir)/gnibbles.2.0.scores
+	-chgrp games $(DESTDIR)$(scoredir)/gnibbles.2.0.scores
 	-chmod 664 $(DESTDIR)$(scoredir)/gnibbles.2.0.scores
 	-touch $(DESTDIR)$(scoredir)/gnibbles.1.0.scores
-	-chown games.games $(DESTDIR)$(scoredir)/gnibbles.1.0.scores
+	-chgrp games $(DESTDIR)$(scoredir)/gnibbles.1.0.scores
 	-chmod 664 $(DESTDIR)$(scoredir)/gnibbles.1.0.scores
 	-touch $(DESTDIR)$(scoredir)/gnibbles.4.1.scores
-	-chown games.games $(DESTDIR)$(scoredir)/gnibbles.4.1.scores
+	-chgrp games $(DESTDIR)$(scoredir)/gnibbles.4.1.scores
 	-chmod 664 $(DESTDIR)$(scoredir)/gnibbles.4.1.scores
 	-touch $(DESTDIR)$(scoredir)/gnibbles.3.1.scores
-	-chown games.games $(DESTDIR)$(scoredir)/gnibbles.3.1.scores
+	-chgrp games $(DESTDIR)$(scoredir)/gnibbles.3.1.scores
 	-chmod 664 $(DESTDIR)$(scoredir)/gnibbles.3.1.scores
 	-touch $(DESTDIR)$(scoredir)/gnibbles.2.1.scores
-	-chown games.games $(DESTDIR)$(scoredir)/gnibbles.2.1.scores
+	-chgrp games $(DESTDIR)$(scoredir)/gnibbles.2.1.scores
 	-chmod 664 $(DESTDIR)$(scoredir)/gnibbles.2.1.scores
 	-touch $(DESTDIR)$(scoredir)/gnibbles.1.1.scores
-	-chown games.games $(DESTDIR)$(scoredir)/gnibbles.1.1.scores
+	-chgrp games $(DESTDIR)$(scoredir)/gnibbles.1.1.scores
 	-chmod 664 $(DESTDIR)$(scoredir)/gnibbles.1.1.scores
-	-chgrp games $(DESTDIR)$(bindir)/gnibbles && chmod 2111 $(DESTDIR)$(bindir)/gnibbles
+	-chgrp games $(DESTDIR)$(bindir)/gnibbles && chmod 2755 $(DESTDIR)$(bindir)/gnibbles
--- gnome-games-1.4.0.4/gnobots2/Makefile.am.orig	Fri Jul 13 00:56:09 2001
+++ gnome-games-1.4.0.4/gnobots2/Makefile.am	Sat May 25 10:54:35 2002
@@ -90,8 +90,8 @@
             fname1=$(DESTDIR)$(scoredir)/gnobots2.$$cfn2.scores; \
             fname2=$(DESTDIR)$(scoredir)/gnobots2.$$cfn2-safe.scores; \
             fname3=$(DESTDIR)$(scoredir)/gnobots2.$$cfn2-super-safe.scores; \
-	    touch $$fname1; chmod 664 $$fname1; chown games:games $$fname1; \
-	    touch $$fname2; chmod 664 $$fname2; chown games:games $$fname2; \
-	    touch $$fname3; chmod 664 $$fname3; chown games:games $$fname3; \
+	    touch $$fname1; chmod 664 $$fname1; chgrp games $$fname1; \
+	    touch $$fname2; chmod 664 $$fname2; chgrp games $$fname2; \
+	    touch $$fname3; chmod 664 $$fname3; chgrp games $$fname3; \
 	    done
-	-chgrp games $(DESTDIR)$(bindir)/gnobots2 && chmod 2111 $(DESTDIR)$(bindir)/gnobots2
+	-chgrp games $(DESTDIR)$(bindir)/gnobots2 && chmod 2755 $(DESTDIR)$(bindir)/gnobots2
--- gnome-games-1.4.0.org/gnome-stones/Makefile.am	Fri Mar 23 16:21:56 2001
+++ gnome-games-1.4.0/gnome-stones/Makefile.am	Fri Mar 23 16:22:27 2001
@@ -67,9 +67,9 @@
 install-data-local:
 	-$(mkinstalldirs) $(DESTDIR)$(scoredir)
 	-touch $(DESTDIR)$(scoredir)/gnome-stones.scores
-	-chown games.games $(DESTDIR)$(scoredir)/gnome-stones.scores
+	-chgrp games $(DESTDIR)$(scoredir)/gnome-stones.scores
 	-chmod 664 $(DESTDIR)$(scoredir)/gnome-stones.scores	
-	-chgrp games $(DESTDIR)$(bindir)/gnome-stones && chmod 2111 $(DESTDIR)$(bindir)/gnome-stones
+	-chgrp games $(DESTDIR)$(bindir)/gnome-stones && chmod 2755 $(DESTDIR)$(bindir)/gnome-stones
 
 gnome-stones.keys: gnome-stones.keys.in Makefile
 	sed -e 's^\@icondir\@^$(Pixmapdir)^g' < $(srcdir)/gnome-stones.keys.in > gnome-stones.keys.tmp \
diff -urN gnome-games-1.4.0.org/gnometris/Makefile.am gnome-games-1.4.0/gnometris/Makefile.am
--- gnome-games-1.4.0.org/gnometris/Makefile.am	Fri Mar 23 16:21:57 2001
+++ gnome-games-1.4.0/gnometris/Makefile.am	Fri Mar 23 16:22:27 2001
@@ -38,13 +38,6 @@
 install-data-local:
 	-$(mkinstalldirs) $(DESTDIR)$(scoredir)
 	touch $(DESTDIR)$(scoredir)/gnometris.scores
-	-chown games.games $(DESTDIR)$(scoredir)/gnometris.scores
+	-chgrp games $(DESTDIR)$(scoredir)/gnometris.scores
 	-chmod 664 $(DESTDIR)$(scoredir)/gnometris.scores
-	-chgrp games $(DESTDIR)$(bindir)/gnometris && chmod 2111 $(DESTDIR)$(bindir)/gnometris
-
-
-
-
-
-
-
+	-chgrp games $(DESTDIR)$(bindir)/gnometris && chmod 2755 $(DESTDIR)$(bindir)/gnometris
diff -urN gnome-games-1.4.0.org/gnomine/Makefile.am gnome-games-1.4.0/gnomine/Makefile.am
--- gnome-games-1.4.0.org/gnomine/Makefile.am	Fri Mar 23 16:21:53 2001
+++ gnome-games-1.4.0/gnomine/Makefile.am	Fri Mar 23 16:22:27 2001
@@ -27,15 +27,15 @@
 install-data-local:
 	-$(mkinstalldirs) $(DESTDIR)$(scoredir)
 	-touch $(DESTDIR)$(scoredir)/gnomine.Tiny.scores
-	-chown games.games $(DESTDIR)$(scoredir)/gnomine.Tiny.scores
+	-chgrp games $(DESTDIR)$(scoredir)/gnomine.Tiny.scores
 	-chmod 664 $(DESTDIR)$(scoredir)/gnomine.Tiny.scores	
 	-touch $(DESTDIR)$(scoredir)/gnomine.Medium.scores
-	-chown games.games $(DESTDIR)$(scoredir)/gnomine.Medium.scores
+	-chgrp games $(DESTDIR)$(scoredir)/gnomine.Medium.scores
 	-chmod 664 $(DESTDIR)$(scoredir)/gnomine.Medium.scores	
 	-touch $(DESTDIR)$(scoredir)/gnomine.Biiiig.scores
-	-chown games.games $(DESTDIR)$(scoredir)/gnomine.Biiiig.scores
+	-chgrp games $(DESTDIR)$(scoredir)/gnomine.Biiiig.scores
 	-chmod 664 $(DESTDIR)$(scoredir)/gnomine.Biiiig.scores	
 	-touch $(DESTDIR)$(scoredir)/gnomine.Custom.scores
-	-chown games.games $(DESTDIR)$(scoredir)/gnomine.Custom.scores
+	-chgrp games $(DESTDIR)$(scoredir)/gnomine.Custom.scores
 	-chmod 664 $(DESTDIR)$(scoredir)/gnomine.Custom.scores	
-	-chgrp games $(DESTDIR)$(bindir)/gnomine && chmod 2111 $(DESTDIR)$(bindir)/gnomine
+	-chgrp games $(DESTDIR)$(bindir)/gnomine && chmod 2755 $(DESTDIR)$(bindir)/gnomine
diff -urN gnome-games-1.4.0.org/gnotravex/Makefile.am gnome-games-1.4.0/gnotravex/Makefile.am
--- gnome-games-1.4.0.org/gnotravex/Makefile.am	Fri Mar 23 16:21:56 2001
+++ gnome-games-1.4.0/gnotravex/Makefile.am	Fri Mar 23 16:22:27 2001
@@ -25,18 +25,18 @@
 install-data-local:
 	-$(mkinstalldirs) $(DESTDIR)$(scoredir)
 	-touch $(DESTDIR)$(scoredir)/gnotravex.2x2.scores
-	-chown games.games $(DESTDIR)$(scoredir)/gnotravex.2x2.scores
+	-chgrp games $(DESTDIR)$(scoredir)/gnotravex.2x2.scores
 	-chmod 664 $(DESTDIR)$(scoredir)/gnotravex.2x2.scores
 	-touch $(DESTDIR)$(scoredir)/gnotravex.3x3.scores
-	-chown games.games $(DESTDIR)$(scoredir)/gnotravex.3x3.scores
+	-chgrp games $(DESTDIR)$(scoredir)/gnotravex.3x3.scores
 	-chmod 664 $(DESTDIR)$(scoredir)/gnotravex.3x3.scores
 	-touch $(DESTDIR)$(scoredir)/gnotravex.4x4.scores
-	-chown games.games $(DESTDIR)$(scoredir)/gnotravex.4x4.scores
+	-chgrp games $(DESTDIR)$(scoredir)/gnotravex.4x4.scores
 	-chmod 664 $(DESTDIR)$(scoredir)/gnotravex.4x4.scores
 	-touch $(DESTDIR)$(scoredir)/gnotravex.5x5.scores
-	-chown games.games $(DESTDIR)$(scoredir)/gnotravex.5x5.scores
+	-chgrp games $(DESTDIR)$(scoredir)/gnotravex.5x5.scores
 	-chmod 664 $(DESTDIR)$(scoredir)/gnotravex.5x5.scores
 	-touch $(DESTDIR)$(scoredir)/gnotravex.6x6.scores
-	-chown games.games $(DESTDIR)$(scoredir)/gnotravex.6x6.scores
+	-chgrp games $(DESTDIR)$(scoredir)/gnotravex.6x6.scores
 	-chmod 664 $(DESTDIR)$(scoredir)/gnotravex.6x6.scores
-	-chgrp games $(DESTDIR)$(bindir)/gnotravex && chmod 2111 $(DESTDIR)$(bindir)/gnotravex
+	-chgrp games $(DESTDIR)$(bindir)/gnotravex && chmod 2755 $(DESTDIR)$(bindir)/gnotravex
--- gnome-games-1.4.0.3/gnotski/Makefile.am.orig	Sat Jul  7 08:13:07 2001
+++ gnome-games-1.4.0.3/gnotski/Makefile.am	Thu Aug  9 18:14:31 2001
-scoredir=$(DESTDIR)$(localstatedir)/games
+scoredir=$(localstatedir)/games
 pixmapdir=$(datadir)/pixmaps
 Gamesdir=$(datadir)/gnome/apps/Games
 
@@ -26,64 +26,64 @@
 
 install-data-local:
 	-$(mkinstalldirs) $(DESTDIR)$(scoredir)
-	-chgrp games $(DESTDIR)$(bindir)/gnotski && chmod 2111 $(DESTDIR)$(bindir)/gnotski
-	-touch             $(scoredir)/gnotski.1.scores
-	-chown games.games $(scoredir)/gnotski.1.scores
-	-chmod 664         $(scoredir)/gnotski.1.scores
-	-touch             $(scoredir)/gnotski.2.scores
-	-chown games.games $(scoredir)/gnotski.2.scores
-	-chmod 664         $(scoredir)/gnotski.2.scores
-	-touch             $(scoredir)/gnotski.3.scores
-	-chown games.games $(scoredir)/gnotski.3.scores
-	-chmod 664         $(scoredir)/gnotski.3.scores
-	-touch             $(scoredir)/gnotski.4.scores
-	-chown games.games $(scoredir)/gnotski.4.scores
-	-chmod 664         $(scoredir)/gnotski.4.scores
-	-touch             $(scoredir)/gnotski.5.scores
-	-chown games.games $(scoredir)/gnotski.5.scores
-	-chmod 664         $(scoredir)/gnotski.5.scores
-	-touch             $(scoredir)/gnotski.6.scores
-	-chown games.games $(scoredir)/gnotski.6.scores
-	-chmod 664         $(scoredir)/gnotski.6.scores
-	-touch             $(scoredir)/gnotski.7.scores
-	-chown games.games $(scoredir)/gnotski.7.scores
-	-chmod 664         $(scoredir)/gnotski.7.scores
-	-touch             $(scoredir)/gnotski.11.scores
-	-chown games.games $(scoredir)/gnotski.11.scores
-	-chmod 664         $(scoredir)/gnotski.11.scores
-	-touch             $(scoredir)/gnotski.12.scores
-	-chown games.games $(scoredir)/gnotski.12.scores
-	-chmod 664         $(scoredir)/gnotski.12.scores
-	-touch             $(scoredir)/gnotski.13.scores
-	-chown games.games $(scoredir)/gnotski.13.scores
-	-chmod 664         $(scoredir)/gnotski.13.scores
-	-touch             $(scoredir)/gnotski.14.scores
-	-chown games.games $(scoredir)/gnotski.14.scores
-	-chmod 664         $(scoredir)/gnotski.14.scores
-	-touch             $(scoredir)/gnotski.15.scores
-	-chown games.games $(scoredir)/gnotski.15.scores
-	-chmod 664         $(scoredir)/gnotski.15.scores
-	-touch             $(scoredir)/gnotski.16.scores
-	-chown games.games $(scoredir)/gnotski.16.scores
-	-chmod 664         $(scoredir)/gnotski.16.scores
-	-touch             $(scoredir)/gnotski.17.scores
-	-chown games.games $(scoredir)/gnotski.17.scores
-	-chmod 664         $(scoredir)/gnotski.17.scores
-	-touch             $(scoredir)/gnotski.21.scores
-	-chown games.games $(scoredir)/gnotski.21.scores
-	-chmod 664         $(scoredir)/gnotski.21.scores
-	-touch             $(scoredir)/gnotski.22.scores
-	-chown games.games $(scoredir)/gnotski.22.scores
-	-chmod 664         $(scoredir)/gnotski.22.scores
-	-touch             $(scoredir)/gnotski.23.scores
-	-chown games.games $(scoredir)/gnotski.23.scores
-	-chmod 664         $(scoredir)/gnotski.23.scores
-	-touch             $(scoredir)/gnotski.24.scores
-	-chown games.games $(scoredir)/gnotski.24.scores
-	-chmod 664         $(scoredir)/gnotski.24.scores
-	-touch             $(scoredir)/gnotski.25.scores
-	-chown games.games $(scoredir)/gnotski.25.scores
-	-chmod 664         $(scoredir)/gnotski.25.scores
-	-touch             $(scoredir)/gnotski.26.scores
-	-chown games.games $(scoredir)/gnotski.26.scores
-	-chmod 664         $(scoredir)/gnotski.26.scores
+	-chgrp games $(DESTDIR)$(bindir)/gnotski && chmod 2755 $(DESTDIR)$(bindir)/gnotski
+	-touch $(DESTDIR)$(scoredir)/gnotski.1.scores
+	-chgrp games $(DESTDIR)$(scoredir)/gnotski.1.scores
+	-chmod 664 $(DESTDIR)$(scoredir)/gnotski.1.scores
+	-touch $(DESTDIR)$(scoredir)/gnotski.2.scores
+	-chgrp games $(DESTDIR)$(scoredir)/gnotski.2.scores
+	-chmod 664 $(DESTDIR)$(scoredir)/gnotski.2.scores
+	-touch $(DESTDIR)$(scoredir)/gnotski.3.scores
+	-chgrp games $(DESTDIR)$(scoredir)/gnotski.3.scores
+	-chmod 664 $(DESTDIR)$(scoredir)/gnotski.3.scores
+	-touch $(DESTDIR)$(scoredir)/gnotski.4.scores
+	-chgrp games $(DESTDIR)$(scoredir)/gnotski.4.scores
+	-chmod 664 $(DESTDIR)$(scoredir)/gnotski.4.scores
+	-touch $(DESTDIR)$(scoredir)/gnotski.5.scores
+	-chgrp games $(DESTDIR)$(scoredir)/gnotski.5.scores
+	-chmod 664 $(DESTDIR)$(scoredir)/gnotski.5.scores
+	-touch $(DESTDIR)$(scoredir)/gnotski.6.scores
+	-chgrp games $(DESTDIR)$(scoredir)/gnotski.6.scores
+	-chmod 664 $(DESTDIR)$(scoredir)/gnotski.6.scores
+	-touch $(DESTDIR)$(scoredir)/gnotski.7.scores
+	-chgrp games $(DESTDIR)$(scoredir)/gnotski.7.scores
+	-chmod 664 $(DESTDIR)$(scoredir)/gnotski.7.scores
+	-touch $(DESTDIR)$(scoredir)/gnotski.11.scores
+	-chgrp games $(DESTDIR)$(scoredir)/gnotski.11.scores
+	-chmod 664 $(DESTDIR)$(scoredir)/gnotski.11.scores
+	-touch $(DESTDIR)$(scoredir)/gnotski.12.scores
+	-chgrp games $(DESTDIR)$(scoredir)/gnotski.12.scores
+	-chmod 664 $(DESTDIR)$(scoredir)/gnotski.12.scores
+	-touch $(DESTDIR)$(scoredir)/gnotski.13.scores
+	-chgrp games $(DESTDIR)$(scoredir)/gnotski.13.scores
+	-chmod 664 $(DESTDIR)$(scoredir)/gnotski.13.scores
+	-touch $(DESTDIR)$(scoredir)/gnotski.14.scores
+	-chgrp games $(DESTDIR)$(scoredir)/gnotski.14.scores
+	-chmod 664 $(DESTDIR)$(scoredir)/gnotski.14.scores
+	-touch $(DESTDIR)$(scoredir)/gnotski.15.scores
+	-chgrp games $(DESTDIR)$(scoredir)/gnotski.15.scores
+	-chmod 664 $(DESTDIR)$(scoredir)/gnotski.15.scores
+	-touch $(DESTDIR)$(scoredir)/gnotski.16.scores
+	-chgrp games $(DESTDIR)$(scoredir)/gnotski.16.scores
+	-chmod 664 $(DESTDIR)$(scoredir)/gnotski.16.scores
+	-touch $(DESTDIR)$(scoredir)/gnotski.17.scores
+	-chgrp games $(DESTDIR)$(scoredir)/gnotski.17.scores
+	-chmod 664 $(DESTDIR)$(scoredir)/gnotski.17.scores
+	-touch $(DESTDIR)$(scoredir)/gnotski.21.scores
+	-chgrp games $(DESTDIR)$(scoredir)/gnotski.21.scores
+	-chmod 664 $(DESTDIR)$(scoredir)/gnotski.21.scores
+	-touch $(DESTDIR)$(scoredir)/gnotski.22.scores
+	-chgrp games $(DESTDIR)$(scoredir)/gnotski.22.scores
+	-chmod 664 $(DESTDIR)$(scoredir)/gnotski.22.scores
+	-touch $(DESTDIR)$(scoredir)/gnotski.23.scores
+	-chgrp games $(DESTDIR)$(scoredir)/gnotski.23.scores
+	-chmod 664 $(DESTDIR)$(scoredir)/gnotski.23.scores
+	-touch $(DESTDIR)$(scoredir)/gnotski.24.scores
+	-chgrp games $(DESTDIR)$(scoredir)/gnotski.24.scores
+	-chmod 664 $(DESTDIR)$(scoredir)/gnotski.24.scores
+	-touch $(DESTDIR)$(scoredir)/gnotski.25.scores
+	-chgrp games $(DESTDIR)$(scoredir)/gnotski.25.scores
+	-chmod 664 $(DESTDIR)$(scoredir)/gnotski.25.scores
+	-touch $(DESTDIR)$(scoredir)/gnotski.26.scores
+	-chgrp games $(DESTDIR)$(scoredir)/gnotski.26.scores
+	-chmod 664 $(DESTDIR)$(scoredir)/gnotski.26.scores
diff -urN gnome-games-1.4.0.org/gtali/Makefile.am gnome-games-1.4.0/gtali/Makefile.am
--- gnome-games-1.4.0.org/gtali/Makefile.am	Fri Mar 23 16:21:55 2001
+++ gnome-games-1.4.0/gtali/Makefile.am	Fri Mar 23 16:22:27 2001
@@ -53,6 +53,6 @@
 install-data-local:
 	-$(mkinstalldirs) $(DESTDIR)$(scoredir)
 	-touch $(DESTDIR)$(scoredir)/gtali.scores
-	-chown games.games $(DESTDIR)$(scoredir)/gtali.scores
+	-chgrp games $(DESTDIR)$(scoredir)/gtali.scores
 	-chmod 664 $(DESTDIR)$(scoredir)/gtali.scores	
-	-chgrp games $(DESTDIR)$(bindir)/gtali && chmod 2111 $(DESTDIR)$(bindir)/gtali
+	-chgrp games $(DESTDIR)$(bindir)/gtali && chmod 2755 $(DESTDIR)$(bindir)/gtali
diff -urN gnome-games-1.4.0.org/iagno/Makefile.am gnome-games-1.4.0/iagno/Makefile.am
--- gnome-games-1.4.0.org/iagno/Makefile.am	Fri Mar 23 16:21:55 2001
+++ gnome-games-1.4.0/iagno/Makefile.am	Fri Mar 23 16:22:27 2001
@@ -67,19 +67,19 @@
 	touch $(DESTDIR)$(scoredir)/iagno.b1.scores
 	touch $(DESTDIR)$(scoredir)/iagno.b2.scores
 	touch $(DESTDIR)$(scoredir)/iagno.b3.scores
-	-chown games.games $(DESTDIR)$(scoredir)/iagno.w1.scores
-	-chown games.games $(DESTDIR)$(scoredir)/iagno.w2.scores
-	-chown games.games $(DESTDIR)$(scoredir)/iagno.w3.scores
-	-chown games.games $(DESTDIR)$(scoredir)/iagno.b1.scores
-	-chown games.games $(DESTDIR)$(scoredir)/iagno.b2.scores
-	-chown games.games $(DESTDIR)$(scoredir)/iagno.b3.scores
+	-chgrp games $(DESTDIR)$(scoredir)/iagno.w1.scores
+	-chgrp games $(DESTDIR)$(scoredir)/iagno.w2.scores
+	-chgrp games $(DESTDIR)$(scoredir)/iagno.w3.scores
+	-chgrp games $(DESTDIR)$(scoredir)/iagno.b1.scores
+	-chgrp games $(DESTDIR)$(scoredir)/iagno.b2.scores
+	-chgrp games $(DESTDIR)$(scoredir)/iagno.b3.scores
 	-chmod 664 $(DESTDIR)$(scoredir)/iagno.w1.scores
 	-chmod 664 $(DESTDIR)$(scoredir)/iagno.w2.scores
 	-chmod 664 $(DESTDIR)$(scoredir)/iagno.w3.scores
 	-chmod 664 $(DESTDIR)$(scoredir)/iagno.b1.scores
 	-chmod 664 $(DESTDIR)$(scoredir)/iagno.b2.scores
 	-chmod 664 $(DESTDIR)$(scoredir)/iagno.b3.scores
-	-chgrp games $(DESTDIR)$(bindir)/iagno && chmod 2111 $(DESTDIR)$(bindir)/iagno
+	-chgrp games $(DESTDIR)$(bindir)/iagno && chmod 2755 $(DESTDIR)$(bindir)/iagno
 
 soundlistdir = $(sysconfdir)/sound/events
 soundlist_DATA = iagno.soundlist
diff -urN gnome-games-1.4.0.org/mahjongg/Makefile.am gnome-games-1.4.0/mahjongg/Makefile.am
--- gnome-games-1.4.0.org/mahjongg/Makefile.am	Fri Mar 23 16:21:54 2001
+++ gnome-games-1.4.0/mahjongg/Makefile.am	Fri Mar 23 16:22:27 2001
@@ -39,10 +39,7 @@
 	-$(mkinstalldirs) $(DESTDIR)$(scoredir)
 	$(mkinstalldirs) $(DESTDIR)$(bgdir)
 	touch $(DESTDIR)$(scoredir)/mahjongg.{easy,difficult}.scores
-	-chown games.games $(DESTDIR)$(scoredir)/mahjongg.{easy,difficult}.scores
+	-chgrp games $(DESTDIR)$(scoredir)/mahjongg.{easy,difficult}.scores
 	-chmod 664 $(DESTDIR)$(scoredir)/mahjongg.{easy,difficult}.scores
-	-chgrp games $(DESTDIR)$(bindir)/mahjongg && chmod 2111 $(DESTDIR)$(bindir)/mahjongg
+	-chgrp games $(DESTDIR)$(bindir)/mahjongg && chmod 2755 $(DESTDIR)$(bindir)/mahjongg
 	$(mkinstalldirs) $(DESTDIR)$(Gamesdir)
-
-
-
diff -urN gnome-games-1.4.0.org/same-gnome/Makefile.am gnome-games-1.4.0/same-gnome/Makefile.am
--- gnome-games-1.4.0.org/same-gnome/Makefile.am	Fri Mar 23 16:21:54 2001
+++ gnome-games-1.4.0/same-gnome/Makefile.am	Fri Mar 23 16:22:27 2001
@@ -28,6 +28,6 @@
 install-data-local:
 	-$(mkinstalldirs) $(DESTDIR)$(scoredir)
 	touch $(DESTDIR)$(scoredir)/same-gnome.scores
-	-chown games.games $(DESTDIR)$(scoredir)/same-gnome.scores
+	-chgrp games $(DESTDIR)$(scoredir)/same-gnome.scores
 	-chmod 664 $(DESTDIR)$(scoredir)/same-gnome.scores
-	-chgrp games $(DESTDIR)$(bindir)/same-gnome && chmod 2111 $(DESTDIR)$(bindir)/same-gnome
+	-chgrp games $(DESTDIR)$(bindir)/same-gnome && chmod 2755 $(DESTDIR)$(bindir)/same-gnome
diff -urN gnome-games-1.4.0.org/sgmldocs.make gnome-games-1.4.0/sgmldocs.make
--- gnome-games-1.4.0.org/sgmldocs.make	Fri Mar 23 16:21:52 2001
+++ gnome-games-1.4.0/sgmldocs.make	Fri Mar 23 16:25:26 2001
@@ -58,7 +58,7 @@
 	-cp $(docname)/index.html .
 
 $(docname).sgml: $(sgml_ents)
-        -ourdir=`pwd`;  \
+        ourdir=`pwd`;  \
         cd $(srcdir);   \
         cp $(sgml_ents) $$ourdir
 
diff -urN gnome-games-1.4.0.org/xbill/Makefile.am gnome-games-1.4.0/xbill/Makefile.am
--- gnome-games-1.4.0.org/xbill/Makefile.am	Fri Mar 23 16:21:58 2001
+++ gnome-games-1.4.0/xbill/Makefile.am	Fri Mar 23 16:22:27 2001
@@ -27,7 +27,6 @@
 install-data-local:
 	$(mkinstalldirs) $(DESTDIR)$(scoredir)
 	-touch $(DESTDIR)$(scoredir)/gnome-xbill.scores
-	-chown games.games $(DESTDIR)$(scoredir)/gnome-xbill.scores
+	-chgrp games $(DESTDIR)$(scoredir)/gnome-xbill.scores
 	-chmod 664 $(DESTDIR)$(scoredir)/gnome-xbill.scores
-	-chgrp games $(DESTDIR)$(bindir)/gnome-xbill && chmod 2111 $(DESTDIR)$(bindir)/gnome-xbill
-
+	-chgrp games $(DESTDIR)$(bindir)/gnome-xbill && chmod 2755 $(DESTDIR)$(bindir)/gnome-xbill
--- gnome-games-1.4.0.3/aisleriot/Makefile.am~	Wed Jul 11 21:40:18 2001
+++ gnome-games-1.4.0.3/aisleriot/Makefile.am	Fri Sep 21 07:57:01 2001
@@ -1,7 +1,6 @@
 scoredir=$(localstatedir)/games
 cardimagedir = $(datadir)/pixmaps/cards
 icondir = $(datadir)/pixmaps
-Gamesdir  = $(datadir)/gnome/apps/Games
 typesdir  = $(datadir)/sol-games
 
 SUBDIRS=docs pixmaps
@@ -125,6 +124,6 @@
 	template.scm translatable_game_names.h
 
 
-sysdir = $(datadir)/gnome/apps/Games
-sys_DATA = sol.desktop
+Gamesdir = $(datadir)/gnome/apps/Games
+Games_DATA = sol.desktop
 
--- gnome-games-1.4.0.3/gnotski/Makefile.am~	Fri May 24 19:00:35 2002
+++ gnome-games-1.4.0.3/gnotski/Makefile.am	Fri May 24 19:00:35 2002
@@ -1,4 +1,4 @@
-scoredir=$(DESTDIR)$(localstatedir)/games
+scoredir=$(localstatedir)/games
 pixmapdir=$(datadir)/pixmaps
 Gamesdir=$(datadir)/applnk/Games
 
