diff -urN aa/gnome-games-2.5.6/configure.in gnome-games-2.5.6/configure.in
--- aa/gnome-games-2.5.6/configure.in	2004-02-16 07:46:14.000000000 +0100
+++ gnome-games-2.5.6/configure.in	2004-02-16 13:30:08.548242552 +0100
@@ -84,7 +84,7 @@
 GETTEXT_PACKAGE=gnome-games
 AC_SUBST(GETTEXT_PACKAGE)
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE,"$GETTEXT_PACKAGE", [The gettext package name])
-ALL_LINGUAS="am ar az be bg bn ca cs cy da de el en_GB es et eu fa fi fr ga gl he hi hr hu id it ja ko lt lv mk ml mn ms nl nn no pl pt pt_BR ro ru sk sl sq sr sr@Latn sv tr uk vi wa zh_CN zh_TW"
+ALL_LINGUAS="am ar az be bg bn ca cs cy da de el en_GB es et eu fa fi fr ga gl he hi hr hu id it ja ko lt lv mk ml mn ms nb nl nn pl pt pt_BR ro ru sk sl sq sr sr@Latn sv tr uk vi wa zh_CN zh_TW"
 AM_GLIB_GNU_GETTEXT
 # Since there is controversy over using it, we will check for it.
 AC_CHECK_FUNCS(ngettext)
