diff -urN gnome-games-2.22.1/m4/ggz.m4 gnome-games-2.22.1.new/m4/ggz.m4
--- gnome-games-2.22.1/m4/ggz.m4	2008-04-07 05:42:16.000000000 +0200
+++ gnome-games-2.22.1.new/m4/ggz.m4	2008-04-07 10:47:33.000000000 +0200
@@ -1018,7 +1018,7 @@
       AC_GGZ_GGZDMOD([ggz_server="yes"], [ggz_server="no"])
     fi
     if test "$ggz_server" = "yes"; then
-      AC_GGZ_SERVER
+      AC_GGZ_SERVER([], [force])
       AC_DEFINE(GGZ_SERVER, 1, [Server support for GGZ])
     else
       if test "$try_ggz_server" = "yes"; then
