diff -urN gnome-games-2.17.92/glchess/glchess.desktop.in gnome-games-2.17.92.new/glchess/glchess.desktop.in
--- gnome-games-2.17.92/glchess/glchess.desktop.in	2007-02-12 22:49:40.000000000 +0100
+++ gnome-games-2.17.92.new/glchess/glchess.desktop.in	2007-03-08 21:55:40.000000000 +0100
@@ -6,7 +6,7 @@
 Exec=glchess
 Terminal=false
 Type=Application
-Categories=GNOME;Game;BoardGame;
+Categories=GNOME;GTK;Game;BoardGame;
 StartupNotify=true
 Icon=glchess
 MimeType=application/x-chess-pgn;
diff -urN gnome-games-2.17.92/gnome-sudoku/gnome-sudoku.desktop.in gnome-games-2.17.92.new/gnome-sudoku/gnome-sudoku.desktop.in
--- gnome-games-2.17.92/gnome-sudoku/gnome-sudoku.desktop.in	2007-02-12 22:50:21.000000000 +0100
+++ gnome-games-2.17.92.new/gnome-sudoku/gnome-sudoku.desktop.in	2007-03-08 21:56:32.000000000 +0100
@@ -6,7 +6,7 @@
 Exec=gnome-sudoku
 Terminal=false
 Type=Application
-Categories=GNOME;Game;LogicGame;
+Categories=GNOME;GTK;Game;LogicGame;
 StartupNotify=true
 Icon=sudoku.png
 Name[en_GB]=Sudoku
