diff -urN aaa/gnome-games-2.3.7/gtali/gtali.schemas.in gnome-games-2.3.7/gtali/gtali.schemas.in
--- aaa/gnome-games-2.3.7/gtali/gtali.schemas.in	2003-08-10 08:06:24.000000000 +0200
+++ gnome-games-2.3.7/gtali/gtali.schemas.in	2003-08-10 18:33:27.000000000 +0200
@@ -54,10 +54,10 @@
       <owner>gtali</owner>
       <type>list</type>
       <list_type>string</list_type>
+      <default>[Human,Wilber,Bill,Monica,Kenneth,Janet]</default>
       <locale name="C">
-        <default>[Human,Wilber,Bill,Monica,Kenneth,Janet]</default>
-        <short></short>
-        <long></long>
+        <short>Player names</short>
+        <long>Player names.</long>
       </locale>
     </schema>
 
