diff -urN gnome-games-2.1.3.orig/freecell/src/Makefile.in gnome-games-2.1.3/freecell/src/Makefile.in
--- gnome-games-2.1.3.orig/freecell/src/Makefile.in	Mon Nov 25 21:39:34 2002
+++ gnome-games-2.1.3/freecell/src/Makefile.in	Mon Nov 25 21:54:43 2002
@@ -454,7 +454,6 @@
 @INTLTOOL_SCHEMAS_RULE@
 
 install-data-local:
-	GCONF_CONFIG_SOURCE=$(GCONF_SCHEMA_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(srcdir)/$(schema_DATA)
 
 # Tell versions [3.59,3.63) of GNU make to not export all variables.
 # Otherwise a system limit (for SysV at least) may be exceeded.
diff -urN gnome-games-2.1.3.orig/gataxx/Makefile.in gnome-games-2.1.3/gataxx/Makefile.in
--- gnome-games-2.1.3.orig/gataxx/Makefile.in	Mon Nov 25 21:39:32 2002
+++ gnome-games-2.1.3/gataxx/Makefile.in	Mon Nov 25 21:55:45 2002
@@ -594,7 +594,6 @@
 @INTLTOOL_SOUNDLIST_RULE@
 
 install-data-local:
-	GCONF_CONFIG_SOURCE=$(GCONF_SCHEMA_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(srcdir)/$(schema_DATA)
 
 # Tell versions [3.59,3.63) of GNU make to not export all variables.
 # Otherwise a system limit (for SysV at least) may be exceeded.
diff -urN gnome-games-2.1.3.orig/gnometris/Makefile.in gnome-games-2.1.3/gnometris/Makefile.in
--- gnome-games-2.1.3.orig/gnometris/Makefile.in	Mon Nov 25 21:39:34 2002
+++ gnome-games-2.1.3/gnometris/Makefile.in	Mon Nov 25 21:56:27 2002
@@ -536,7 +536,6 @@
 @INTLTOOL_SCHEMAS_RULE@
 
 install-data-local:
-	GCONF_CONFIG_SOURCE=$(GCONF_SCHEMA_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(srcdir)/$(schema_DATA)
 	-$(mkinstalldirs) $(DESTDIR)$(scoredir)
 	touch $(DESTDIR)$(scoredir)/gnometris.scores
 	-chown games.games $(DESTDIR)$(scoredir)/gnometris.scores
diff -urN gnome-games-2.1.3.orig/gnomine/Makefile.in gnome-games-2.1.3/gnomine/Makefile.in
--- gnome-games-2.1.3.orig/gnomine/Makefile.in	Mon Nov 25 21:39:31 2002
+++ gnome-games-2.1.3/gnomine/Makefile.in	Mon Nov 25 21:56:41 2002
@@ -540,7 +540,6 @@
 @INTLTOOL_DESKTOP_RULE@
 
 install-data-local:
-	GCONF_CONFIG_SOURCE=$(GCONF_SCHEMA_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(srcdir)/$(schema_DATA)
 	-$(mkinstalldirs) $(DESTDIR)$(scoredir)
 	-touch $(DESTDIR)$(scoredir)/gnomine.Tiny.scores
 	-chown games.games $(DESTDIR)$(scoredir)/gnomine.Tiny.scores
diff -urN gnome-games-2.1.3.orig/gtali/Makefile.in gnome-games-2.1.3/gtali/Makefile.in
--- gnome-games-2.1.3.orig/gtali/Makefile.in	Mon Nov 25 21:39:32 2002
+++ gnome-games-2.1.3/gtali/Makefile.in	Mon Nov 25 21:57:39 2002
@@ -540,7 +540,6 @@
 @INTLTOOL_DESKTOP_RULE@
 
 install-data-local:
-	-GCONF_CONFIG_SOURCE=$(GCONF_SCHEMA_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(srcdir)/$(schema_DATA)
 	-$(mkinstalldirs) $(DESTDIR)$(scoredir)
 	-touch $(DESTDIR)$(scoredir)/gtali.scores
 	-chown games.games $(DESTDIR)$(scoredir)/gtali.scores
diff -urN gnome-games-2.1.3.orig/iagno/Makefile.in gnome-games-2.1.3/iagno/Makefile.in
--- gnome-games-2.1.3.orig/iagno/Makefile.in	Mon Nov 25 21:39:33 2002
+++ gnome-games-2.1.3/iagno/Makefile.in	Mon Nov 25 21:57:32 2002
@@ -595,7 +595,6 @@
 @INTLTOOL_SCHEMAS_RULE@
 
 install-data-local:
-	GCONF_CONFIG_SOURCE=$(GCONF_SCHEMA_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(srcdir)/$(schema_DATA)
 	-$(mkinstalldirs) $(DESTDIR)$(scoredir)
 	touch $(DESTDIR)$(scoredir)/iagno.w1.scores
 	touch $(DESTDIR)$(scoredir)/iagno.w2.scores
diff -urN gnome-games-2.1.3.orig/mahjongg/Makefile.in gnome-games-2.1.3/mahjongg/Makefile.in
--- gnome-games-2.1.3.orig/mahjongg/Makefile.in	Mon Nov 25 21:39:31 2002
+++ gnome-games-2.1.3/mahjongg/Makefile.in	Mon Nov 25 21:57:44 2002
@@ -591,7 +591,6 @@
 @INTLTOOL_DESKTOP_RULE@
 
 install-data-local:
-	GCONF_CONFIG_SOURCE=$(GCONF_SCHEMA_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(srcdir)/$(schema_DATA)
 	-$(mkinstalldirs) $(DESTDIR)$(scoredir)
 	$(mkinstalldirs) $(DESTDIR)$(bgdir)
 	touch $(DESTDIR)$(scoredir)/mahjongg.{easy,difficult}.scores
diff -urN gnome-games-2.1.3.orig/same-gnome/Makefile.in gnome-games-2.1.3/same-gnome/Makefile.in
--- gnome-games-2.1.3.orig/same-gnome/Makefile.in	Mon Nov 25 21:39:31 2002
+++ gnome-games-2.1.3/same-gnome/Makefile.in	Mon Nov 25 21:57:49 2002
@@ -536,7 +536,6 @@
 @INTLTOOL_DESKTOP_RULE@
 
 install-data-local:
-	GCONF_CONFIG_SOURCE=$(GCONF_SCHEMA_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(srcdir)/$(schema_DATA)
 	-$(mkinstalldirs) $(DESTDIR)$(scoredir)
 	touch $(DESTDIR)$(scoredir)/same-gnome.scores
 	-chown games.games $(DESTDIR)$(scoredir)/same-gnome.scores
