diff -urN gnome-games-2.0.0.orig/freecell/src/Makefile.in gnome-games-2.0.0/freecell/src/Makefile.in
--- gnome-games-2.0.0.orig/freecell/src/Makefile.in	Tue Jun 18 08:40:32 2002
+++ gnome-games-2.0.0/freecell/src/Makefile.in	Tue Jun 18 09:06:15 2002
@@ -451,7 +451,6 @@
 @INTLTOOL_DESKTOP_RULE@
 
 install-data-local:
-	GCONF_CONFIG_SOURCE=$(GCONF_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(srcdir)/$(schema_DATA)
 
 # Tell versions [3.59,3.63) of GNU make to not export all variables.
 # Otherwise a system limit (for SysV at least) may be exceeded.
diff -urN gnome-games-2.0.0.orig/gataxx/Makefile.in gnome-games-2.0.0/gataxx/Makefile.in
--- gnome-games-2.0.0.orig/gataxx/Makefile.in	Tue Jun 18 08:40:32 2002
+++ gnome-games-2.0.0/gataxx/Makefile.in	Tue Jun 18 08:55:03 2002
@@ -588,7 +588,7 @@
 @INTLTOOL_SOUNDLIST_RULE@
 
 install-data-local:
-	GCONF_CONFIG_SOURCE=$(GCONF_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(srcdir)/$(schema_DATA)
+
 
 # Tell versions [3.59,3.63) of GNU make to not export all variables.
 # Otherwise a system limit (for SysV at least) may be exceeded.
diff -urN gnome-games-2.0.0.orig/gnometris/Makefile.in gnome-games-2.0.0/gnometris/Makefile.in
--- gnome-games-2.0.0.orig/gnometris/Makefile.in	Tue Jun 18 08:40:31 2002
+++ gnome-games-2.0.0/gnometris/Makefile.in	Tue Jun 18 08:56:12 2002
@@ -530,7 +530,6 @@
 @INTLTOOL_DESKTOP_RULE@
 
 install-data-local:
-	GCONF_CONFIG_SOURCE=$(GCONF_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(srcdir)/$(schema_DATA)
 	-$(mkinstalldirs) $(DESTDIR)$(scoredir)
 	touch $(DESTDIR)$(scoredir)/gnometris.scores
 	-chown games.games $(DESTDIR)$(scoredir)/gnometris.scores
diff -urN gnome-games-2.0.0.orig/gnomine/Makefile.in gnome-games-2.0.0/gnomine/Makefile.in
--- gnome-games-2.0.0.orig/gnomine/Makefile.in	Tue Jun 18 08:40:32 2002
+++ gnome-games-2.0.0/gnomine/Makefile.in	Tue Jun 18 08:56:21 2002
@@ -534,7 +534,6 @@
 @INTLTOOL_DESKTOP_RULE@
 
 install-data-local:
-	GCONF_CONFIG_SOURCE=$(GCONF_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(srcdir)/$(schema_DATA)
 	-$(mkinstalldirs) $(DESTDIR)$(scoredir)
 	-touch $(DESTDIR)$(scoredir)/gnomine.Tiny.scores
 	-chown games.games $(DESTDIR)$(scoredir)/gnomine.Tiny.scores
diff -urN gnome-games-2.0.0.orig/gtali/Makefile.in gnome-games-2.0.0/gtali/Makefile.in
--- gnome-games-2.0.0.orig/gtali/Makefile.in	Tue Jun 18 08:40:31 2002
+++ gnome-games-2.0.0/gtali/Makefile.in	Tue Jun 18 08:56:47 2002
@@ -534,7 +534,6 @@
 @INTLTOOL_DESKTOP_RULE@
 
 install-data-local:
-	-GCONF_CONFIG_SOURCE=$(GCONF_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(srcdir)/$(schema_DATA)
 	-$(mkinstalldirs) $(DESTDIR)$(scoredir)
 	-touch $(DESTDIR)$(scoredir)/gtali.scores
 	-chown games.games $(DESTDIR)$(scoredir)/gtali.scores
diff -urN gnome-games-2.0.0.orig/iagno/Makefile.in gnome-games-2.0.0/iagno/Makefile.in
--- gnome-games-2.0.0.orig/iagno/Makefile.in	Tue Jun 18 08:40:31 2002
+++ gnome-games-2.0.0/iagno/Makefile.in	Tue Jun 18 08:56:57 2002
@@ -589,7 +589,6 @@
 @INTLTOOL_DESKTOP_RULE@
 
 install-data-local:
-	GCONF_CONFIG_SOURCE=$(GCONF_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(srcdir)/$(schema_DATA)
 	-$(mkinstalldirs) $(DESTDIR)$(scoredir)
 	touch $(DESTDIR)$(scoredir)/iagno.w1.scores
 	touch $(DESTDIR)$(scoredir)/iagno.w2.scores
diff -urN gnome-games-2.0.0.orig/mahjongg/Makefile.in gnome-games-2.0.0/mahjongg/Makefile.in
--- gnome-games-2.0.0.orig/mahjongg/Makefile.in	Tue Jun 18 08:40:32 2002
+++ gnome-games-2.0.0/mahjongg/Makefile.in	Tue Jun 18 08:57:42 2002
@@ -585,7 +585,6 @@
 @INTLTOOL_DESKTOP_RULE@
 
 install-data-local:
-	GCONF_CONFIG_SOURCE=$(GCONF_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(srcdir)/$(schema_DATA)
 	-$(mkinstalldirs) $(DESTDIR)$(scoredir)
 	$(mkinstalldirs) $(DESTDIR)$(bgdir)
 	touch $(DESTDIR)$(scoredir)/mahjongg.{easy,difficult}.scores
diff -urN gnome-games-2.0.0.orig/same-gnome/Makefile.in gnome-games-2.0.0/same-gnome/Makefile.in
--- gnome-games-2.0.0.orig/same-gnome/Makefile.in	Tue Jun 18 08:40:32 2002
+++ gnome-games-2.0.0/same-gnome/Makefile.in	Tue Jun 18 08:57:52 2002
@@ -530,7 +530,6 @@
 @INTLTOOL_DESKTOP_RULE@
 
 install-data-local:
-	GCONF_CONFIG_SOURCE=$(GCONF_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(srcdir)/$(schema_DATA)
 	-$(mkinstalldirs) $(DESTDIR)$(scoredir)
 	touch $(DESTDIR)$(scoredir)/same-gnome.scores
 	-chown games.games $(DESTDIR)$(scoredir)/same-gnome.scores
--- gnome-games-2.0.0/gnotski/Makefile.in.wiget	Tue Jun 18 17:27:05 2002
+++ gnome-games-2.0.0/gnotski/Makefile.in	Tue Jun 18 17:28:42 2002
@@ -131,7 +131,7 @@
 WARN_CFLAGS = @WARN_CFLAGS@
 ac_cv_prog_cxx_works = @ac_cv_prog_cxx_works@
 
-scoredir = $(DESTDIR)$(localstatedir)/games
+scoredir = $(localstatedir)/games
 pixmapdir = $(datadir)/pixmaps
 Gamesdir = $(datadir)/applications
 
@@ -431,66 +431,66 @@
 install-data-local:
 	-$(mkinstalldirs) $(DESTDIR)$(scoredir)
 	-chgrp games $(DESTDIR)$(bindir)/gnotski && chmod 2111 $(DESTDIR)$(bindir)/gnotski
-	-touch             $(scoredir)/gnotski.1.scores
-	-chown games.games $(scoredir)/gnotski.1.scores
-	-chmod 664         $(scoredir)/gnotski.1.scores
-	-touch             $(scoredir)/gnotski.2.scores
-	-chown games.games $(scoredir)/gnotski.2.scores
-	-chmod 664         $(scoredir)/gnotski.2.scores
-	-touch             $(scoredir)/gnotski.3.scores
-	-chown games.games $(scoredir)/gnotski.3.scores
-	-chmod 664         $(scoredir)/gnotski.3.scores
-	-touch             $(scoredir)/gnotski.4.scores
-	-chown games.games $(scoredir)/gnotski.4.scores
-	-chmod 664         $(scoredir)/gnotski.4.scores
-	-touch             $(scoredir)/gnotski.5.scores
-	-chown games.games $(scoredir)/gnotski.5.scores
-	-chmod 664         $(scoredir)/gnotski.5.scores
-	-touch             $(scoredir)/gnotski.6.scores
-	-chown games.games $(scoredir)/gnotski.6.scores
-	-chmod 664         $(scoredir)/gnotski.6.scores
-	-touch             $(scoredir)/gnotski.7.scores
-	-chown games.games $(scoredir)/gnotski.7.scores
-	-chmod 664         $(scoredir)/gnotski.7.scores
-	-touch             $(scoredir)/gnotski.11.scores
-	-chown games.games $(scoredir)/gnotski.11.scores
-	-chmod 664         $(scoredir)/gnotski.11.scores
-	-touch             $(scoredir)/gnotski.12.scores
-	-chown games.games $(scoredir)/gnotski.12.scores
-	-chmod 664         $(scoredir)/gnotski.12.scores
-	-touch             $(scoredir)/gnotski.13.scores
-	-chown games.games $(scoredir)/gnotski.13.scores
-	-chmod 664         $(scoredir)/gnotski.13.scores
-	-touch             $(scoredir)/gnotski.14.scores
-	-chown games.games $(scoredir)/gnotski.14.scores
-	-chmod 664         $(scoredir)/gnotski.14.scores
-	-touch             $(scoredir)/gnotski.15.scores
-	-chown games.games $(scoredir)/gnotski.15.scores
-	-chmod 664         $(scoredir)/gnotski.15.scores
-	-touch             $(scoredir)/gnotski.16.scores
-	-chown games.games $(scoredir)/gnotski.16.scores
-	-chmod 664         $(scoredir)/gnotski.16.scores
-	-touch             $(scoredir)/gnotski.17.scores
-	-chown games.games $(scoredir)/gnotski.17.scores
-	-chmod 664         $(scoredir)/gnotski.17.scores
-	-touch             $(scoredir)/gnotski.21.scores
-	-chown games.games $(scoredir)/gnotski.21.scores
-	-chmod 664         $(scoredir)/gnotski.21.scores
-	-touch             $(scoredir)/gnotski.22.scores
-	-chown games.games $(scoredir)/gnotski.22.scores
-	-chmod 664         $(scoredir)/gnotski.22.scores
-	-touch             $(scoredir)/gnotski.23.scores
-	-chown games.games $(scoredir)/gnotski.23.scores
-	-chmod 664         $(scoredir)/gnotski.23.scores
-	-touch             $(scoredir)/gnotski.24.scores
-	-chown games.games $(scoredir)/gnotski.24.scores
-	-chmod 664         $(scoredir)/gnotski.24.scores
-	-touch             $(scoredir)/gnotski.25.scores
-	-chown games.games $(scoredir)/gnotski.25.scores
-	-chmod 664         $(scoredir)/gnotski.25.scores
-	-touch             $(scoredir)/gnotski.26.scores
-	-chown games.games $(scoredir)/gnotski.26.scores
-	-chmod 664         $(scoredir)/gnotski.26.scores
+	-touch             $(DESTDIR)$(scoredir)/gnotski.1.scores
+	-chown games.games $(DESTDIR)$(scoredir)/gnotski.1.scores
+	-chmod 664         $(DESTDIR)$(scoredir)/gnotski.1.scores
+	-touch             $(DESTDIR)$(scoredir)/gnotski.2.scores
+	-chown games.games $(DESTDIR)$(scoredir)/gnotski.2.scores
+	-chmod 664         $(DESTDIR)$(scoredir)/gnotski.2.scores
+	-touch             $(DESTDIR)$(scoredir)/gnotski.3.scores
+	-chown games.games $(DESTDIR)$(scoredir)/gnotski.3.scores
+	-chmod 664         $(DESTDIR)$(scoredir)/gnotski.3.scores
+	-touch             $(DESTDIR)$(scoredir)/gnotski.4.scores
+	-chown games.games $(DESTDIR)$(scoredir)/gnotski.4.scores
+	-chmod 664         $(DESTDIR)$(scoredir)/gnotski.4.scores
+	-touch             $(DESTDIR)$(scoredir)/gnotski.5.scores
+	-chown games.games $(DESTDIR)$(scoredir)/gnotski.5.scores
+	-chmod 664         $(DESTDIR)$(scoredir)/gnotski.5.scores
+	-touch             $(DESTDIR)$(scoredir)/gnotski.6.scores
+	-chown games.games $(DESTDIR)$(scoredir)/gnotski.6.scores
+	-chmod 664         $(DESTDIR)$(scoredir)/gnotski.6.scores
+	-touch             $(DESTDIR)$(scoredir)/gnotski.7.scores
+	-chown games.games $(DESTDIR)$(scoredir)/gnotski.7.scores
+	-chmod 664         $(DESTDIR)$(scoredir)/gnotski.7.scores
+	-touch             $(DESTDIR)$(scoredir)/gnotski.11.scores
+	-chown games.games $(DESTDIR)$(scoredir)/gnotski.11.scores
+	-chmod 664         $(DESTDIR)$(scoredir)/gnotski.11.scores
+	-touch             $(DESTDIR)$(scoredir)/gnotski.12.scores
+	-chown games.games $(DESTDIR)$(scoredir)/gnotski.12.scores
+	-chmod 664         $(DESTDIR)$(scoredir)/gnotski.12.scores
+	-touch             $(DESTDIR)$(scoredir)/gnotski.13.scores
+	-chown games.games $(DESTDIR)$(scoredir)/gnotski.13.scores
+	-chmod 664         $(DESTDIR)$(scoredir)/gnotski.13.scores
+	-touch             $(DESTDIR)$(scoredir)/gnotski.14.scores
+	-chown games.games $(DESTDIR)$(scoredir)/gnotski.14.scores
+	-chmod 664         $(DESTDIR)$(scoredir)/gnotski.14.scores
+	-touch             $(DESTDIR)$(scoredir)/gnotski.15.scores
+	-chown games.games $(DESTDIR)$(scoredir)/gnotski.15.scores
+	-chmod 664         $(DESTDIR)$(scoredir)/gnotski.15.scores
+	-touch             $(DESTDIR)$(scoredir)/gnotski.16.scores
+	-chown games.games $(DESTDIR)$(scoredir)/gnotski.16.scores
+	-chmod 664         $(DESTDIR)$(scoredir)/gnotski.16.scores
+	-touch             $(DESTDIR)$(scoredir)/gnotski.17.scores
+	-chown games.games $(DESTDIR)$(scoredir)/gnotski.17.scores
+	-chmod 664         $(DESTDIR)$(scoredir)/gnotski.17.scores
+	-touch             $(DESTDIR)$(scoredir)/gnotski.21.scores
+	-chown games.games $(DESTDIR)$(scoredir)/gnotski.21.scores
+	-chmod 664         $(DESTDIR)$(scoredir)/gnotski.21.scores
+	-touch             $(DESTDIR)$(scoredir)/gnotski.22.scores
+	-chown games.games $(DESTDIR)$(scoredir)/gnotski.22.scores
+	-chmod 664         $(DESTDIR)$(scoredir)/gnotski.22.scores
+	-touch             $(DESTDIR)$(scoredir)/gnotski.23.scores
+	-chown games.games $(DESTDIR)$(scoredir)/gnotski.23.scores
+	-chmod 664         $(DESTDIR)$(scoredir)/gnotski.23.scores
+	-touch             $(DESTDIR)$(scoredir)/gnotski.24.scores
+	-chown games.games $(DESTDIR)$(scoredir)/gnotski.24.scores
+	-chmod 664         $(DESTDIR)$(scoredir)/gnotski.24.scores
+	-touch             $(DESTDIR)$(scoredir)/gnotski.25.scores
+	-chown games.games $(DESTDIR)$(scoredir)/gnotski.25.scores
+	-chmod 664         $(DESTDIR)$(scoredir)/gnotski.25.scores
+	-touch             $(DESTDIR)$(scoredir)/gnotski.26.scores
+	-chown games.games $(DESTDIR)$(scoredir)/gnotski.26.scores
+	-chmod 664         $(DESTDIR)$(scoredir)/gnotski.26.scores
 
 # Tell versions [3.59,3.63) of GNU make to not export all variables.
 # Otherwise a system limit (for SysV at least) may be exceeded.
--- gnome-games-2.0.0/aisleriot/Makefile.in.wiget2	Tue Jun 18 17:31:35 2002
+++ gnome-games-2.0.0/aisleriot/Makefile.in	Tue Jun 18 17:32:52 2002
@@ -160,7 +160,7 @@
 EXTRA_DIST = AUTHORS $(types_DATA) $(cardimage_DATA) 	$(icon_DATA) sol.desktop.in SCHEME_GAMES.h 	template.scm translatable_game_names.h
 
 
-sysdir = $(datadir)/gnome/apps/Games
+sysdir = $(Gamesdir)
 sys_in_files = sol.desktop.in
 sys_DATA = $(sys_in_files:.desktop.in=.desktop)
 mkinstalldirs = $(SHELL) $(top_srcdir)/mkinstalldirs
