diff -urN aaa/gnome-games-2.4.1.1/gnibbles/gnibbles.desktop.in gnome-games-2.4.1.1/gnibbles/gnibbles.desktop.in
--- aaa/gnome-games-2.4.1.1/gnibbles/gnibbles.desktop.in	1997-01-01 06:23:13.000000000 +0100
+++ gnome-games-2.4.1.1/gnibbles/gnibbles.desktop.in	2003-12-12 18:19:12.000000000 +0100
@@ -6,7 +6,6 @@
 Icon=gnome-nibbles.png
 Terminal=false
 Type=Application
-Icon=gnome-nibbles.png
 Categories=GNOME;Application;Game;ArcadeGame;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-gammes
