diff -urN gnome-games-2.0.3.orig/freecell/src/Makefile.in gnome-games-2.0.3/freecell/src/Makefile.in
--- gnome-games-2.0.3.orig/freecell/src/Makefile.in	Fri Aug 16 11:06:34 2002
+++ gnome-games-2.0.3/freecell/src/Makefile.in	Fri Aug 16 11:14:23 2002
@@ -435,7 +435,6 @@
 @INTLTOOL_DESKTOP_RULE@
 
 install-data-local:
-	GCONF_CONFIG_SOURCE=$(GCONF_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(srcdir)/$(schema_DATA)

 # Tell versions [3.59,3.63) of GNU make to not export all variables.
 # Otherwise a system limit (for SysV at least) may be exceeded.
diff -urN gnome-games-2.0.3.orig/gataxx/Makefile.in gnome-games-2.0.3/gataxx/Makefile.in
--- gnome-games-2.0.3.orig/gataxx/Makefile.in	Fri Aug 16 11:06:34 2002
+++ gnome-games-2.0.3/gataxx/Makefile.in	Fri Aug 16 11:14:43 2002
@@ -582,7 +582,6 @@
 @INTLTOOL_SOUNDLIST_RULE@
 
 install-data-local:
-	GCONF_CONFIG_SOURCE=$(GCONF_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(srcdir)/$(schema_DATA)

 # Tell versions [3.59,3.63) of GNU make to not export all variables.
 # Otherwise a system limit (for SysV at least) may be exceeded.
 .NOEXPORT:
diff -urN gnome-games-2.0.3.orig/gnometris/Makefile.in gnome-games-2.0.3/gnometris/Makefile.in
--- gnome-games-2.0.3.orig/gnometris/Makefile.in	Fri Aug 16 11:06:34 2002
+++ gnome-games-2.0.3/gnometris/Makefile.in	Fri Aug 16 11:15:27 2002
@@ -611,7 +611,6 @@
 @INTLTOOL_DESKTOP_RULE@
 
 install-data-local:
-	GCONF_CONFIG_SOURCE=$(GCONF_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(srcdir)/$(schema_DATA)
 	-$(mkinstalldirs) $(DESTDIR)$(scoredir)
 	touch $(DESTDIR)$(scoredir)/gnometris.scores
 	-chown games.games $(DESTDIR)$(scoredir)/gnometris.scores
diff -urN gnome-games-2.0.3.orig/gnomine/Makefile.in gnome-games-2.0.3/gnomine/Makefile.in
--- gnome-games-2.0.3.orig/gnomine/Makefile.in	Fri Aug 16 11:06:34 2002
+++ gnome-games-2.0.3/gnomine/Makefile.in	Fri Aug 16 11:15:46 2002
@@ -586,7 +586,6 @@
 @INTLTOOL_DESKTOP_RULE@
 
 install-data-local:
-	GCONF_CONFIG_SOURCE=$(GCONF_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(srcdir)/$(schema_DATA)
 	-$(mkinstalldirs) $(DESTDIR)$(scoredir)
 	-touch $(DESTDIR)$(scoredir)/gnomine.Tiny.scores
 	-chown games.games $(DESTDIR)$(scoredir)/gnomine.Tiny.scores
diff -urN gnome-games-2.0.3.orig/gnotski/Makefile.in gnome-games-2.0.3/gnotski/Makefile.in
--- gnome-games-2.0.3.orig/gnotski/Makefile.in	Fri Aug 16 11:06:34 2002
+++ gnome-games-2.0.3/gnotski/Makefile.in	Fri Aug 16 11:23:40 2002
@@ -430,66 +430,66 @@
 install-data-local:
 	-$(mkinstalldirs) $(DESTDIR)$(scoredir)
 	-chgrp games $(DESTDIR)$(bindir)/gnotski && chmod 2555 $(DESTDIR)$(bindir)/gnotski
-	-touch             $(scoredir)/gnotski.1.scores
-	-chown games.games $(scoredir)/gnotski.1.scores
-	-chmod 664         $(scoredir)/gnotski.1.scores
-	-touch             $(scoredir)/gnotski.2.scores
-	-chown games.games $(scoredir)/gnotski.2.scores
-	-chmod 664         $(scoredir)/gnotski.2.scores
-	-touch             $(scoredir)/gnotski.3.scores
-	-chown games.games $(scoredir)/gnotski.3.scores
-	-chmod 664         $(scoredir)/gnotski.3.scores
-	-touch             $(scoredir)/gnotski.4.scores
-	-chown games.games $(scoredir)/gnotski.4.scores
-	-chmod 664         $(scoredir)/gnotski.4.scores
-	-touch             $(scoredir)/gnotski.5.scores
-	-chown games.games $(scoredir)/gnotski.5.scores
-	-chmod 664         $(scoredir)/gnotski.5.scores
-	-touch             $(scoredir)/gnotski.6.scores
-	-chown games.games $(scoredir)/gnotski.6.scores
-	-chmod 664         $(scoredir)/gnotski.6.scores
-	-touch             $(scoredir)/gnotski.7.scores
-	-chown games.games $(scoredir)/gnotski.7.scores
-	-chmod 664         $(scoredir)/gnotski.7.scores
-	-touch             $(scoredir)/gnotski.11.scores
-	-chown games.games $(scoredir)/gnotski.11.scores
-	-chmod 664         $(scoredir)/gnotski.11.scores
-	-touch             $(scoredir)/gnotski.12.scores
-	-chown games.games $(scoredir)/gnotski.12.scores
-	-chmod 664         $(scoredir)/gnotski.12.scores
-	-touch             $(scoredir)/gnotski.13.scores
-	-chown games.games $(scoredir)/gnotski.13.scores
-	-chmod 664         $(scoredir)/gnotski.13.scores
-	-touch             $(scoredir)/gnotski.14.scores
-	-chown games.games $(scoredir)/gnotski.14.scores
-	-chmod 664         $(scoredir)/gnotski.14.scores
-	-touch             $(scoredir)/gnotski.15.scores
-	-chown games.games $(scoredir)/gnotski.15.scores
-	-chmod 664         $(scoredir)/gnotski.15.scores
-	-touch             $(scoredir)/gnotski.16.scores
-	-chown games.games $(scoredir)/gnotski.16.scores
-	-chmod 664         $(scoredir)/gnotski.16.scores
-	-touch             $(scoredir)/gnotski.17.scores
-	-chown games.games $(scoredir)/gnotski.17.scores
-	-chmod 664         $(scoredir)/gnotski.17.scores
-	-touch             $(scoredir)/gnotski.21.scores
-	-chown games.games $(scoredir)/gnotski.21.scores
-	-chmod 664         $(scoredir)/gnotski.21.scores
-	-touch             $(scoredir)/gnotski.22.scores
-	-chown games.games $(scoredir)/gnotski.22.scores
-	-chmod 664         $(scoredir)/gnotski.22.scores
-	-touch             $(scoredir)/gnotski.23.scores
-	-chown games.games $(scoredir)/gnotski.23.scores
-	-chmod 664         $(scoredir)/gnotski.23.scores
-	-touch             $(scoredir)/gnotski.24.scores
-	-chown games.games $(scoredir)/gnotski.24.scores
-	-chmod 664         $(scoredir)/gnotski.24.scores
-	-touch             $(scoredir)/gnotski.25.scores
-	-chown games.games $(scoredir)/gnotski.25.scores
-	-chmod 664         $(scoredir)/gnotski.25.scores
-	-touch             $(scoredir)/gnotski.26.scores
-	-chown games.games $(scoredir)/gnotski.26.scores
-	-chmod 664         $(scoredir)/gnotski.26.scores
+	-touch             $(DESTDIR)$(scoredir)/gnotski.1.scores
+	-chown games.games $(DESTDIR)$(scoredir)/gnotski.1.scores
+	-chmod 664         $(DESTDIR)$(scoredir)/gnotski.1.scores
+	-touch             $(DESTDIR)$(scoredir)/gnotski.2.scores
+	-chown games.games $(DESTDIR)$(scoredir)/gnotski.2.scores
+	-chmod 664         $(DESTDIR)$(scoredir)/gnotski.2.scores
+	-touch             $(DESTDIR)$(scoredir)/gnotski.3.scores
+	-chown games.games $(DESTDIR)$(scoredir)/gnotski.3.scores
+	-chmod 664         $(DESTDIR)$(scoredir)/gnotski.3.scores
+	-touch             $(DESTDIR)$(scoredir)/gnotski.4.scores
+	-chown games.games $(DESTDIR)$(scoredir)/gnotski.4.scores
+	-chmod 664         $(DESTDIR)$(scoredir)/gnotski.4.scores
+	-touch             $(DESTDIR)$(scoredir)/gnotski.5.scores
+	-chown games.games $(DESTDIR)$(scoredir)/gnotski.5.scores
+	-chmod 664         $(DESTDIR)$(scoredir)/gnotski.5.scores
+	-touch             $(DESTDIR)$(scoredir)/gnotski.6.scores
+	-chown games.games $(DESTDIR)$(scoredir)/gnotski.6.scores
+	-chmod 664         $(DESTDIR)$(scoredir)/gnotski.6.scores
+	-touch             $(DESTDIR)$(scoredir)/gnotski.7.scores
+	-chown games.games $(DESTDIR)$(scoredir)/gnotski.7.scores
+	-chmod 664         $(DESTDIR)$(scoredir)/gnotski.7.scores
+	-touch             $(DESTDIR)$(scoredir)/gnotski.11.scores
+	-chown games.games $(DESTDIR)$(scoredir)/gnotski.11.scores
+	-chmod 664         $(DESTDIR)$(scoredir)/gnotski.11.scores
+	-touch             $(DESTDIR)$(scoredir)/gnotski.12.scores
+	-chown games.games $(DESTDIR)$(scoredir)/gnotski.12.scores
+	-chmod 664         $(DESTDIR)$(scoredir)/gnotski.12.scores
+	-touch             $(DESTDIR)$(scoredir)/gnotski.13.scores
+	-chown games.games $(DESTDIR)$(scoredir)/gnotski.13.scores
+	-chmod 664         $(DESTDIR)$(scoredir)/gnotski.13.scores
+	-touch             $(DESTDIR)$(scoredir)/gnotski.14.scores
+	-chown games.games $(DESTDIR)$(scoredir)/gnotski.14.scores
+	-chmod 664         $(DESTDIR)$(scoredir)/gnotski.14.scores
+	-touch             $(DESTDIR)$(scoredir)/gnotski.15.scores
+	-chown games.games $(DESTDIR)$(scoredir)/gnotski.15.scores
+	-chmod 664         $(DESTDIR)$(scoredir)/gnotski.15.scores
+	-touch             $(DESTDIR)$(scoredir)/gnotski.16.scores
+	-chown games.games $(DESTDIR)$(scoredir)/gnotski.16.scores
+	-chmod 664         $(DESTDIR)$(scoredir)/gnotski.16.scores
+	-touch             $(DESTDIR)$(scoredir)/gnotski.17.scores
+	-chown games.games $(DESTDIR)$(scoredir)/gnotski.17.scores
+	-chmod 664         $(DESTDIR)$(scoredir)/gnotski.17.scores
+	-touch             $(DESTDIR)$(scoredir)/gnotski.21.scores
+	-chown games.games $(DESTDIR)$(scoredir)/gnotski.21.scores
+	-chmod 664         $(DESTDIR)$(scoredir)/gnotski.21.scores
+	-touch             $(DESTDIR)$(scoredir)/gnotski.22.scores
+	-chown games.games $(DESTDIR)$(scoredir)/gnotski.22.scores
+	-chmod 664         $(DESTDIR)$(scoredir)/gnotski.22.scores
+	-touch             $(DESTDIR)$(scoredir)/gnotski.23.scores
+	-chown games.games $(DESTDIR)$(scoredir)/gnotski.23.scores
+	-chmod 664         $(DESTDIR)$(scoredir)/gnotski.23.scores
+	-touch             $(DESTDIR)$(scoredir)/gnotski.24.scores
+	-chown games.games $(DESTDIR)$(scoredir)/gnotski.24.scores
+	-chmod 664         $(DESTDIR)$(scoredir)/gnotski.24.scores
+	-touch             $(DESTDIR)$(scoredir)/gnotski.25.scores
+	-chown games.games $(DESTDIR)$(scoredir)/gnotski.25.scores
+	-chmod 664         $(DESTDIR)$(scoredir)/gnotski.25.scores
+	-touch             $(DESTDIR)$(scoredir)/gnotski.26.scores
+	-chown games.games $(DESTDIR)$(scoredir)/gnotski.26.scores
+	-chmod 664         $(DESTDIR)$(scoredir)/gnotski.26.scores
 
 # Tell versions [3.59,3.63) of GNU make to not export all variables.
 # Otherwise a system limit (for SysV at least) may be exceeded.
diff -urN gnome-games-2.0.3.orig/gtali/Makefile.in gnome-games-2.0.3/gtali/Makefile.in
--- gnome-games-2.0.3.orig/gtali/Makefile.in	Fri Aug 16 11:06:34 2002
+++ gnome-games-2.0.3/gtali/Makefile.in	Fri Aug 16 11:16:16 2002
@@ -600,7 +600,6 @@
 @INTLTOOL_DESKTOP_RULE@
 
 install-data-local:
-	-GCONF_CONFIG_SOURCE=$(GCONF_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(srcdir)/$(schema_DATA)
 	-$(mkinstalldirs) $(DESTDIR)$(scoredir)
 	-touch $(DESTDIR)$(scoredir)/gtali.scores
 	-chown games.games $(DESTDIR)$(scoredir)/gtali.scores
diff -urN gnome-games-2.0.3.orig/iagno/Makefile.in gnome-games-2.0.3/iagno/Makefile.in
--- gnome-games-2.0.3.orig/iagno/Makefile.in	Fri Aug 16 11:06:34 2002
+++ gnome-games-2.0.3/iagno/Makefile.in	Fri Aug 16 11:16:34 2002
@@ -654,7 +654,6 @@
 @INTLTOOL_DESKTOP_RULE@
 
 install-data-local:
-	GCONF_CONFIG_SOURCE=$(GCONF_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(srcdir)/$(schema_DATA)
 	-$(mkinstalldirs) $(DESTDIR)$(scoredir)
 	touch $(DESTDIR)$(scoredir)/iagno.w1.scores
 	touch $(DESTDIR)$(scoredir)/iagno.w2.scores
diff -urN gnome-games-2.0.3.orig/mahjongg/Makefile.in gnome-games-2.0.3/mahjongg/Makefile.in
--- gnome-games-2.0.3.orig/mahjongg/Makefile.in	Fri Aug 16 11:06:34 2002
+++ gnome-games-2.0.3/mahjongg/Makefile.in	Fri Aug 16 11:16:50 2002
@@ -638,7 +638,6 @@
 @INTLTOOL_DESKTOP_RULE@
 
 install-data-local:
-	GCONF_CONFIG_SOURCE=$(GCONF_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(srcdir)/$(schema_DATA)
 	-$(mkinstalldirs) $(DESTDIR)$(scoredir)
 	$(mkinstalldirs) $(DESTDIR)$(bgdir)
 	touch $(DESTDIR)$(scoredir)/mahjongg.{easy,difficult}.scores
diff -urN gnome-games-2.0.3.orig/same-gnome/Makefile.in gnome-games-2.0.3/same-gnome/Makefile.in
--- gnome-games-2.0.3.orig/same-gnome/Makefile.in	Fri Aug 16 11:06:35 2002
+++ gnome-games-2.0.3/same-gnome/Makefile.in	Fri Aug 16 11:17:20 2002
@@ -578,7 +578,6 @@
 @INTLTOOL_DESKTOP_RULE@
 
 install-data-local:
-	GCONF_CONFIG_SOURCE=$(GCONF_CONFIG_SOURCE) $(GCONFTOOL) --makefile-install-rule $(srcdir)/$(schema_DATA)
 	-$(mkinstalldirs) $(DESTDIR)$(scoredir)
 	touch $(DESTDIR)$(scoredir)/same-gnome.scores
 	-chown games.games $(DESTDIR)$(scoredir)/same-gnome.scores
