--- gnome-games-1.4.0.1/configure.in	Sat Apr 14 04:26:16 2001
+++ gnome-games-1.4.0.1/configure.in~	Wed Mar 28 20:46:07 2001
@@ -56,7 +56,7 @@
 AM_CONDITIONAL(HAS_CURSES, test "$has_curses" = true)
 
 ALL_LINGUAS="cs da de el en_GB es et fi fr ga gl hu it ja ko lt nl no pl pt pt_BR ru sk sv tr uk wa zh_TW.Big5 zh_CN.GB2312"
-AM_GNOME_GETTEXT
+AM_GNU_GETTEXT
 
 AC_SUBST(CFLAGS)
 AC_SUBST(CPPFLAGS)
