diff -urN gnome-games-2.19.91.1/gtali/gtali.schemas.in gnome-games-2.19.91.1.new/gtali/gtali.schemas.in
--- gnome-games-2.19.91.1/gtali/gtali.schemas.in	2007-08-27 22:03:17.000000000 +0200
+++ gnome-games-2.19.91.1.new/gtali/gtali.schemas.in	2007-09-02 21:32:57.000000000 +0200
@@ -54,10 +54,10 @@
       <owner>gtali</owner>
       <type>list</type>
       <list_type>string</list_type>
+      <default>[Human,Wilber,Bill,Monica,Kenneth,Janet]</default>
       <locale name="C">
-        <default>[Human,Wilber,Bill,Monica,Kenneth,Janet]</default>
-        <short></short>
-        <long></long>
+        <short>Player names</short>
+        <long>Player names.</long>
       </locale>
     </schema>
 
@@ -66,10 +66,10 @@
       <applyto>/apps/gtali/GameType</applyto>
       <owner>gtali</owner>
       <type>string</type>
+      <default>Regular</default>
       <locale name="C">
-        <default>Regular</default>
-        <short></short>
-        <long></long>
+        <short>Game type</short>
+        <long>Game type.</long>
       </locale>
     </schema>
 
