--- gnome-games-1.4.0.3/freecell/gdk-card-image/Makefile.am~	Sat Feb 20 22:02:30 1999
+++ gnome-games-1.4.0.3/freecell/gdk-card-image/Makefile.am	Sun Nov 25 09:41:46 2001
@@ -27,7 +27,7 @@
 cardslotdir = $(datadir)/pixmaps/cards/slots/
 cardslot_DATA = hand.png plain.png
 
-INCLUDES = -I$(includedir) $(GNOME_INCLUDEDIR)
+INCLUDES = -I$(includedir) $(GNOME_INCLUDEDIR) $(GDK_PIXBUF_CFLAGS)
 
 EXTRA_DIST = $(cardback_DATA) $(cardhonor_DATA) $(cardjoker_DATA) \
 	$(cardrank_DATA) $(cardsuitsmall_DATA) $(cardsuitlarge_DATA)\
--- gnome-games-1.4.0.3/gnome-stones/objects/Makefile.am~	Tue Nov 10 05:55:56 1998
+++ gnome-games-1.4.0.3/gnome-stones/objects/Makefile.am	Sun Nov 25 10:08:15 2001
@@ -6,7 +6,8 @@
 	-I$(GSTONES_SRC_DIR)				\
 	-DGNOMELOCALEDIR=\""$(datadir)/locale"\" 	\
 	-I$(includedir)					\
-	$(GNOME_INCLUDEDIR)
+	$(GNOME_INCLUDEDIR)				\
+	$(GDK_PIXBUF_CFLAGS)
 
 object_LTLIBRARIES = libgnome-stones.la libgnomekoban.la
 
