--- gnome-games-1.4.0.4/gnome-stones/main.c~	Sat Oct 12 19:24:06 2002
+++ gnome-games-1.4.0.4/gnome-stones/main.c	Sat Oct 12 19:24:06 2002
@@ -1011,7 +1011,7 @@
 static gboolean
 scan_public_plugin_directory (void)
 {
-  char *dir= gnome_unconditional_libdir_file ("gnome-stones/objects/");
+  char *dir= gnome_unconditional_libdir_file ("gnome-stones/");
 
   plugin_load_plugins_in_dir (dir);
 
--- gnome-games-1.4.0.4/gnome-stones/objects/Makefile.am~	Sat Oct 12 13:32:54 2002
+++ gnome-games-1.4.0.4/gnome-stones/objects/Makefile.am	Sat Oct 12 13:32:54 2002
@@ -1,6 +1,6 @@
 GSTONES_SRC_DIR = $(srcdir)/../
 
-objectdir= $(libdir)/gnome-stones/objects
+objectdir= $(libdir)/gnome-stones
 
 INCLUDES = 						\
 	-I$(GSTONES_SRC_DIR)				\
@@ -12,4 +12,7 @@
 object_LTLIBRARIES = libgnome-stones.la libgnomekoban.la
 
 libgnome_stones_la_SOURCES = gnome-stones.c
+libgnome_stones_la_LDFLAGS = -module -avoid-version
+
 libgnomekoban_la_SOURCES   = gnomekoban.c
+libgnomekoban_la_LDFLAGS   = -module -avoid-version
