--- gnome-games-2.6.2/aisleriot/freecell.desktop.in~	2004-08-24 04:49:40.000000000 +0200
+++ gnome-games-2.6.2/aisleriot/freecell.desktop.in	2004-08-24 04:49:40.000000000 +0200
@@ -6,7 +6,7 @@
 Icon=gnome-cardgame.png
 Terminal=false
 Type=Application
-Categories=GNOME;Application;Game;CardGame;
+Categories=GTK;GNOME;Game;CardGame;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-games
 X-GNOME-Bugzilla-Component=aisleriot
--- gnome-games-2.6.2/aisleriot/sol.desktop.in~	2004-08-24 04:49:50.000000000 +0200
+++ gnome-games-2.6.2/aisleriot/sol.desktop.in	2004-08-24 04:49:50.000000000 +0200
@@ -6,7 +6,7 @@
 Icon=gnome-aisleriot.png
 Terminal=false
 Type=Application
-Categories=GNOME;Application;Game;CardGame;
+Categories=GTK;GNOME;Game;CardGame;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-games
 X-GNOME-Bugzilla-Component=aisleriot
--- gnome-games-2.6.2/gataxx/gataxx.desktop.in~	2004-08-24 04:50:07.000000000 +0200
+++ gnome-games-2.6.2/gataxx/gataxx.desktop.in	2004-08-24 04:50:07.000000000 +0200
@@ -6,7 +6,7 @@
 Exec=gataxx
 Terminal=false
 Type=Application
-Categories=GNOME;Application;Game;BoardGame;
+Categories=GTK;GNOME;Game;BoardGame;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-games
 X-GNOME-Bugzilla-Component=gataxx
--- gnome-games-2.6.2/glines/glines.desktop.in~	2004-08-24 04:50:21.000000000 +0200
+++ gnome-games-2.6.2/glines/glines.desktop.in	2004-08-24 04:50:21.000000000 +0200
@@ -6,7 +6,7 @@
 Icon=glines.png
 Terminal=false
 Type=Application
-Categories=GNOME;Application;Game;PuzzleGame;
+Categories=GTK;GNOME;Game;LogicGame;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-games
 X-GNOME-Bugzilla-Component=glines
--- gnome-games-2.6.2/gnibbles/gnibbles.desktop.in~	2004-08-24 04:50:30.000000000 +0200
+++ gnome-games-2.6.2/gnibbles/gnibbles.desktop.in	2004-08-24 04:50:30.000000000 +0200
@@ -6,7 +6,7 @@
 Icon=gnome-nibbles.png
 Terminal=false
 Type=Application
-Categories=GNOME;Application;Game;ArcadeGame;
+Categories=GTK;GNOME;Game;ArcadeGame;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-games
 X-GNOME-Bugzilla-Component=gnibbles
--- gnome-games-2.6.2/gnobots2/gnobots2.desktop.in~	2004-08-24 04:51:33.000000000 +0200
+++ gnome-games-2.6.2/gnobots2/gnobots2.desktop.in	2004-08-24 04:51:33.000000000 +0200
@@ -6,7 +6,7 @@
 Icon=gnome-gnobots2.png
 Terminal=false
 Type=Application
-Categories=GNOME;Application;Game;ArcadeGame;
+Categories=GTK;GNOME;Game;ArcadeGame;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-games
 X-GNOME-Bugzilla-Component=gnobots2
--- gnome-games-2.6.2/gnome-stones/gnome-stones.desktop.in~	2004-08-24 04:51:49.000000000 +0200
+++ gnome-games-2.6.2/gnome-stones/gnome-stones.desktop.in	2004-08-24 04:51:49.000000000 +0200
@@ -6,7 +6,7 @@
 Icon=gnome-stones.png
 Terminal=false
 Type=Application
-Categories=GNOME;Application;Game;PuzzleGame;
+Categories=GTK;GNOME;Game;LogicGame;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-games
 X-GNOME-Bugzilla-Component=gnome-stones
--- gnome-games-2.6.2/gnometris/gnometris.desktop.in~	2004-08-24 04:51:59.000000000 +0200
+++ gnome-games-2.6.2/gnometris/gnometris.desktop.in	2004-08-24 04:51:59.000000000 +0200
@@ -6,7 +6,7 @@
 Exec=gnometris
 Terminal=false
 Type=Application
-Categories=GNOME;Application;Game;BlocksGame;
+Categories=GTK;GNOME;Game;BlocksGame;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-games
 X-GNOME-Bugzilla-Component=gnometris
--- gnome-games-2.6.2/gnomine/gnomine.desktop.in~	2004-08-24 04:52:16.000000000 +0200
+++ gnome-games-2.6.2/gnomine/gnomine.desktop.in	2004-08-24 04:52:16.000000000 +0200
@@ -6,7 +6,7 @@
 Icon=gnome-gnomine.png
 Terminal=false
 Type=Application
-Categories=GNOME;Application;Game;PuzzleGame;
+Categories=GTK;GNOME;Game;LogicGame;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-games
 X-GNOME-Bugzilla-Component=gnomine
--- gnome-games-2.6.2/gnotravex/gnotravex.desktop.in~	2004-08-24 04:52:31.000000000 +0200
+++ gnome-games-2.6.2/gnotravex/gnotravex.desktop.in	2004-08-24 04:52:31.000000000 +0200
@@ -6,7 +6,7 @@
 Icon=gnome-gnotravex.png
 Terminal=false
 Type=Application
-Categories=GNOME;Application;Game;PuzzleGame;
+Categories=GTK;GNOME;Game;LogicGame;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-games
 X-GNOME-Bugzilla-Component=gnotravex
--- gnome-games-2.6.2/gnotski/gnotski.desktop.in~	2004-08-24 04:52:46.000000000 +0200
+++ gnome-games-2.6.2/gnotski/gnotski.desktop.in	2004-08-24 04:52:46.000000000 +0200
@@ -6,7 +6,7 @@
 Icon=gnotski-icon.png
 Terminal=false
 Type=Application
-Categories=GNOME;Application;Game;PuzzleGame;
+Categories=GTK;GNOME;Game;LogicGame;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-games
 X-GNOME-Bugzilla-Component=gnotski
--- gnome-games-2.6.2/gtali/gtali.desktop.in~	2004-08-24 04:54:20.000000000 +0200
+++ gnome-games-2.6.2/gtali/gtali.desktop.in	2004-08-24 04:54:20.000000000 +0200
@@ -6,7 +6,7 @@
 Icon=gnome-gtali.png
 Terminal=false
 Type=Application
-Categories=GNOME;Application;Game;
+Categories=GTK;GNOME;Game;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-games
 X-GNOME-Bugzilla-Component=gtali
--- gnome-games-2.6.2/iagno/iagno.desktop.in~	2004-08-24 05:02:26.000000000 +0200
+++ gnome-games-2.6.2/iagno/iagno.desktop.in	2004-08-24 05:02:26.000000000 +0200
@@ -6,7 +6,7 @@
 Exec=iagno
 Terminal=false
 Type=Application
-Categories=GNOME;Application;Game;
+Categories=GTK;GNOME;GameLogicGame;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-games
 X-GNOME-Bugzilla-Component=iagno
--- gnome-games-2.6.2/mahjongg/mahjongg.desktop.in~	2004-08-24 05:02:36.000000000 +0200
+++ gnome-games-2.6.2/mahjongg/mahjongg.desktop.in	2004-08-24 05:02:36.000000000 +0200
@@ -6,7 +6,7 @@
 Icon=gnome-mahjongg.png
 Terminal=false
 Type=Application
-Categories=GNOME;Application;Game;BoardGame;
+Categories=GTK;GNOME;Game;BoardGame;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-games
 X-GNOME-Bugzilla-Component=mahjongg
--- gnome-games-2.6.2/same-gnome/same-gnome.desktop.in~	2004-08-24 05:02:48.000000000 +0200
+++ gnome-games-2.6.2/same-gnome/same-gnome.desktop.in	2004-08-24 05:02:48.000000000 +0200
@@ -6,7 +6,7 @@
 Icon=gnome-gsame.png
 Terminal=false
 Type=Application
-Categories=GNOME;Application;Game;PuzzleGame;
+Categories=GTK;GNOME;Game;LogicGame;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gnome-games
 X-GNOME-Bugzilla-Component=same-gnome
